### ISSUE TITLE
502 Bad Gateway on recorder.js

### ISSUE THREAD

**Issue by troyschneringer** [smartlook/smartlook-relay-proxy#286]
**Describe the bug**
502 Bad Gateway on `/recorder.js` even though other paths work

**To Reproduce**

1. Deploy Smartlook relay via docker image: `smartlook/relay-proxy:latest`
```
docker run --name="smartlook-relay-proxy" \
  -p 8000:8000 \  # runs on port 8000 by default
  smartlook/relay-proxy:6
```
2. Observe inconsistent respones:
  - 502: http://localhost:8000/recorder.js
  - 200: http://localhost:8000/manager/status

**Expected behavior**
A 200 response for all routes

**Additional context**
Smartlook support directed me here suggesting that my nginx config is somehow incorrect.  However, I am deploying the Smartlook Docker container with no changes so I don't expect that to be the issue.

**Comment by troyschneringer** [comment]
I verified that the latest commit (https://github.com/smartlook/smartlook-relay-proxy/commit/bd2b6b0e56d3ca52ce0375dae0c90cbd724bfcc4) fixes this issue.  Can we get an updated version of the official container with the change?

**Comment by samialdury** [comment]
Hi @troyschneringer,

thank you for opening this issue. I've released [v6.1.0](https://github.com/smartlook/smartlook-relay-proxy/releases/tag/v6.1.0), which includes the fix.

I'm closing this now, feel free to reopen if you encounter any other issues.
